postgres/dev.sls

```diff
@@ -3,12 +3,22 @@
 {% if grains.os not in ('Windows', 'MacOS',) %}
 
   {% if postgres.pkg_dev %}
```

Comment:
Also, by properly setting the scope of `pkg_libpq_dev` for certain operating systems, we could avoid other checks within the state.